src/storage/txn/scheduler.rs

```diff
@@ -48,10 +50,13 @@ use super::Result;
 use super::Error;
 use super::store::SnapshotStore;
 use super::latch::{Latches, Lock};
+use super::statistics::{RegionsWriteStats, ROLL_INTERVAL_SECS, CHECK_EXPIRED_INTERVAL_SECS, U8_MAX};
 use super::super::metrics::*;
 
 // TODO: make it configurable.
 pub const GC_BATCH_SIZE: usize = 512;
+pub const WRITE_STATS_BASE: u64 = 64;
+pub const STATS_EXPIRED_SECS: u64 = 7 * 24 * 3600; // 7 * 24 hours
```

Comment:
When the working set is very large, a GC round may takes a long time, and a region will not receive GC command until next round.